_pytest/python.py

```diff
@@ -1651,6 +1651,21 @@ def _getfuncargvalue(self, fixturedef):
         except (AttributeError, ValueError):
             param = NOTSET
             param_index = 0
+            if fixturedef.params is not None:
+                curdir = py.path.local()
+                frame = inspect.stack()[3]
+                _, source_path, source_lineno, _, _, _ = frame
```

Comment:
This seems kind of weird - if I'm reading the docs correctly you can do `inspect.getframeinfo(frame)` and then have a namedtuple with `filename` and `lineno` attributes.